tests/test_utils.py

```diff
@@ -330,3 +332,36 @@ def test_validate_property_schema(self, helpers):
         actual = validate_utils.validate_property_schema(mock_class)
 
         assert actual is None
+
+
+class TestCsvUtils:
+
+    def test_csv_to_schemaorg(self, helpers, tmp_path):
+        """Test the CSV-to-JSON-LD conversion.
+
+        This test also ensures that the CSV and JSON-LD
+        files for the example data model stay in sync.
+        """
+
+        # instantiate schema explorer
+        base_se = SchemaExplorer()
+
+        # load base schema (BioThings)
+        biothings_path = LOADER.filename("data_models/biothings.model.jsonld")
+        base_se.load_schema(biothings_path)
```

Comment:
Would it be a good idea to refactor this code to use the [`get_schema_explorer()`](https://github.com/Sage-Bionetworks/schematic/blob/develop/tests/test_utils.py#L296) static method from the `Helpers` class?

Reply:
Yeah, absolutely! Do you mind committing that change and merging?